.github/workflows/nightly.yaml

```diff
@@ -16,13 +16,25 @@ on:
   schedule:
     - cron: "0 11 * * 1-5"  # Mon-Fri at 11:00 AM UTC
 
+  # Allow manual invocation – useful for debugging.
+  workflow_dispatch:
```

Comment:
there's no value for this key?

Reply:
That's correct. It doesn't need one unless one wants to pass arguments to the workflow. (In this case, the workflow doesn't take arguments, so there's nothing to add.)

I agree it looks weird :-). But it does work, since it's used in other repos.